brian2/parsing/rendering.py

```diff
@@ -254,6 +255,11 @@ def render_BinOp(self, node):
             op = self.expression_ops['Mult']
             return op(self.render_node(node.left),
                       1 / self.render_node(node.right))
+        elif op_name == 'FloorDiv':
+            op = self.expression_ops['Mult']
+            left = self.render_node(node.left)
+            right = self.render_node(node.right)
+            return sympy.floor(op(left, 1 / right))
```

Comment:
This is a general comment but is it an issue that we convert everything into floats in the SympyRenderer? I had a feeling that we excluded all integer operations from being simplified or operated on by sympy but I can't find that now.

Reply:
The float/integer issue in sympy is ugly, but I think we should postpone this until we do our parsing/rendering refactoring. We have #812 open for that, even though the issue becomes less important with the changes here -- the main problem with having floats instead of integers was that it changed division semantics.

Reply:
OK indeed #812 covers this.